src/en/help-oracle.md

```diff
@@ -2,94 +2,94 @@ Title: Using the Oracle cloud
+    not be creating an instance here.
 
-![Empty Oracle dashboard](./media/oracle_empty-dashboard.png)
+Click on 'Marketplace' on the resulting page (left menu), enter 'ubuntu' into
+the search field, and hit Enter:
 
-Click on the large `Create Instance` tile and select `Compute` from the list of
-services that appear. 
+![Search Ubuntu images](./media/oracle_create-instance-ubuntu-2.png) 
 
-You are now looking at the first step in Oracle's Compute cloud deployment
-process. We'll be using this step to associate images with this specific cloud
-domain before cancelling the remainder of Oracle's cloud deployment. 
+From the point of view of Juju, some images should not be used.
+Juju-compatible images are listed below:
 
-Ubuntu images can be found by clicking on 'Marketplace' from the menu on the
-left and entering `Ubuntu` into the search field that appears. 
+| Version          | Arch   | Series  |
+|------------------| -------|---------|
+| Ubuntu 12.04 LTS | amd64  | Precise |
+| Ubuntu 14.04 LTS | amd64  | Trusty  |
+| Ubuntu 16.04 LTS | amd64  | Xenial  |
+| Ubuntu 17.04     | amd64  | Zesty   |
 
-![Ubuntu image search](./media/oracle_create-instance-ubuntu.png) 
+Since Juju uses charms to install applications, the Ubuntu series you need are
+those that the charms were written for. If unsure, it is recommended to add the
+two most recent LTS releases.
 
-The search will return any official Ubuntu images, visible with the Ubuntu
-logo, alongside other Ubuntu-associated images. 
+!!! Note:
+    At time of writing, Trusty and Xenial are the two most recent Ubuntu
+    LTS releases.
 
-It's currently possible to use any of the following images with Juju:
+Select a compatible image from among the official Ubuntu images (orange Ubuntu
+logo), accept Oracle's terms and conditions, and click 'Install'. Repeat the
+process for each desired image. These installed images will end up under
+'Private Images' in the menu on the left:
 
-| Version          | Arch   | Series  |
-|------------------| -------|---------|
-| Ubuntu 12.04-LTS | amd64  | Precise |
-| Ubuntu 14.04-LTS | amd64  | Trusty  |
-| Ubuntu 16.04-LTS | amd64  | Xenial  |
-| Ubuntu 17.04     | amd64  | Zesty   |
+![List private images](./media/oracle_create-instance-private-2.png)
 
-!!! Warning: 
-	Currently, Oracle's Ubuntu 16.10 (Yakkety Yak) image shouldn't be
-        associated with your Juju/Oracle Compute deployment.
+## Add cloud (trial accounts)
 
-To associate an image, click `Select` at the bottom of the image tile and
-accept Oracle's terms and conditions for associating a Marketplace image with
-your cloud (this needs to be done separately for each image).
+As mentioned, you will need to add your Oracle cloud to Juju if you're using a
+trial account. This requires a 'REST Endpoint'. To get this, navigate to 'My
+Account URL', scroll down to 'Oracle Compute Cloud Service', and click on it.
+The resulting page will look similar to this:
```

Comment:
I tried with a new subscription but things went horribly pear-shaped. I never got an email with the important information. I had to "reset my password" to get in. And then I couldn't continue into Oracle Compute because the system wouldn't let me pass without supplying a mobile phone number different from the one I had used previously, and I only have one cell.

For now I think we should just go with this. I added that the user may need some trial & error to find the correct endpoint (which is just true). BTW, how did you find your endpoint? I see your account has but one and it begins with 'api'. Also, why did you say your used URL is only for paid accounts and then went ahead and used it for the trial account for `add-cloud` (recall that `add-cloud` is not required for paid accounts).